sys/auto_init/netif/auto_init_at86rf2xx.c

```diff
@@ -58,11 +59,19 @@ void auto_init_at86rf2xx(void)
             LOG_ERROR("[auto_init_netif] error initializing at86rf2xx radio #%u\n", i);
         }
         else {
-            gnrc_netdev_init(_at86rf2xx_stacks[i],
-                             AT86RF2XX_MAC_STACKSIZE,
-                             AT86RF2XX_MAC_PRIO,
-                             "at86rf2xx",
-                             &gnrc_adpt[i]);
+#ifdef MODULE_GNRC_LWMAC
+            gnrc_lwmac_init(_at86rf2xx_stacks[i],
+                            AT86RF2XX_MAC_STACKSIZE,
+                            AT86RF2XX_MAC_PRIO,
+                            "at86rf2xx-lwmac",
+                            &gnrc_adpt[i]);
```

Comment:
This was more a note to the community than to you (because I don't see how you could fix that in the context of this PR). Basically what you are saying here: If lwMAC is compiled in, use lwMAC on all interfaces that support it (in this case only interfaces with an at86rf2xx device behind them). That's bad, because a node can have multiple interfaces, and a user might choose to use a different MAC protocol on them (or none at all).